batch/batch/driver/main.py

```diff
@@ -1152,7 +1152,7 @@ async def cancel_fast_failing_batches(app):
 USER_CORES = pc.Gauge('batch_user_cores', 'Batch user cores', ['state', 'user', 'inst_coll'])
 USER_JOBS = pc.Gauge('batch_user_jobs', 'Batch user jobs', ['state', 'user', 'inst_coll'])
 FREE_CORES = pc.Summary('batch_free_cores', 'Batch instance free cores', ['inst_coll'])
-UTILIZATION = pc.Summary('batch_utilization', 'Batch utilization rates', ['inst_coll'])
+TOTAL_CORES = pc.Summary('batch_total_cores', 'Batch instance free cores', ['inst_coll'])
```

Comment:
Can you change the metric description to not be the same as the one above it?

Reply:
whoops, yep